src/main.rs

```diff
@@ -34,16 +37,33 @@ struct Cli {
 
   #[command(flatten)]
   verbose: clap_verbosity_flag::Verbosity,
+
+  /// Controls when to use color.
+  #[arg(
+      long,
+      default_value_t = ColorChoice::Auto,
+      value_name = "WHEN",
+      value_enum,
```

Comment:
is this `value_enum` required, I believe clap can automatically figure that out? also, why do `value_name = "WHEN"`?